topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -485,4 +512,49 @@ The next example is *SRR12416842* from an Indonesia [study](https://www.microbio
 
 There is something clearly wrong with sample SRR12416842, perhaps indicating sample contamination. This example of a sample that doesn't map to the reference genome illustrates that even when sequence quality is good, sequence data problems can become apparent in later steps of analysis and it is important to always have a sense of what results to expect. You can develop a better sense of what quality control results to expect by first practicing techniques with known data before analysing new samples.
 
-We hope you enjoyed this tutorial!
+# Processing many samples at once: Collections and Workflows (optional)
+
+In the tutorial thus far we have focused on processing single samples, where two read datasets (forward and reverse reads) are associated with a single sample. In practice, sequence analysis typically involves analysing batches of samples and we run the same
+analysis steps for each sample in the batch. Galaxy supports working with batches using collections and workflows. 
+
+If you are new to collections, you can learn more about them in [this tutorial]({% link topics/galaxy-interface/tutorials/collections/tutorial.md %}).
+
+If you have followed all of the steps of this tutorial, you will have two samples, one named *004-2* and another *SRR12416842*. Create a collection of lists of pairs from these samples and call the collection `samples`.
+
+> <hands-on-title>Analysing samples with the TB Variant Reporting workflow</hands-on-title>
+>
+> 1. **Import the Workflow** into Galaxy 
```

Comment:
if the workflow is this one in the GTN, there's a snippet you can use that will give folks a two click import experience 

https://training.galaxyproject.org/training-material/news/2023/12/12/tutorial-run-wfh-ds.html

Reply:
That is for workflows in WorkflowHub.eu or Dockstore, right? As of right now the workflow is not in one of those repositories so is this snippet applicable?

Reply:
there is one for grn workflows as well, see the first usage in Minerva tutorial or the assembly large genome tutorial